src/main/java/org/prebid/server/privacy/gdpr/vendorlist/VendorListService.java

```diff
@@ -127,6 +131,15 @@ public VendorListService(String cacheDir,
             return Future.succeededFuture(fallbackVendorList);
         }
 
+        if (deprecated) {
```

Comment:
Don't think we need this condition, rather i'd recommend to modify `shouldFallback(..)` to take into account `deprecated` flag.